### PR TITLE
Vanga desactivate

### PR DESCRIPTION
**Pull request by DedrickEnc**
This PR gives the possibility of locking a user. It closes https://github.com/Vanga-Hospital/bhima-2.X/issues/260.

----
Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through JSHint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

File: client/src/partials/users/user.modal.html [hunk 1]
```diff
@@ -60,6 +60,18 @@
     </div>
 
     <div ng-if="!UserModalCtrl.isCreating" class="form-group">
+      <label class="control-label">
+        <input
+          type="checkbox"
+          name="is_active"
```
Comment (jniles) [inline client/src/partials/users/user.modal.html:66]: It is a bit confusing that `is_active` is the name of the `<input>` (which means that `is_active=1` is an active user?), but the model is `deactivated` (which means that `deactivated=1` is an inactive user?).

File: server/lib/errors/Unauthorized.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ function Unauthorized(description) {
   this.status = 401;
 
   // bhima status code (for $translation)
-  this.code = 'ERRORS.UNAUTHORIZED';
+  this.code = code || 'ERRORS.UNAUTHORIZED';
```
Comment (jniles) [inline server/lib/errors/Unauthorized.js:42]: Good improvement to this library :+1:

File: client/src/partials/users/user.modal.html [hunk 3]
```diff
@@ -60,6 +60,18 @@
...
+          ng-true-value="1"
+          ng-false-value="0"
+          ng-model="UserModalCtrl.user.deactivated">
+        {{ "FORM.LABELS.LOCKED" | translate }}
```
Comment (jniles) [inline client/src/partials/users/user.modal.html:70]: Try to make it a common practice to use the translate directive...

**Comment by DedrickEnc** [comment]
@jniles @sfount

**Pull request closed without merging**